src/main/usage/service.ts

```diff
@@ -1,76 +1,77 @@
 import PhotosSubmodule from '@internxt/sdk/dist/photos/photos';
-import fetch, { HeadersInit } from 'electron-fetch';
 import Logger from 'electron-log';
+import { getClient } from '../../shared/HttpClient/main-process-client';
 
-import { getHeaders, obtainToken } from '../auth/service';
+import { obtainToken } from '../auth/service';
 import { Usage } from './usage';
 
 const driveUrl = process.env.API_URL;
 const photosUrl = process.env.PHOTOS_URL;
 
+const httpClient = getClient();
+
 async function getPhotosUsage(): Promise<number> {
-	if (!photosUrl) {
-		throw new Error('PHOTOS API URL NOT DEFINED');
-	}
+  if (!photosUrl) {
+    throw new Error('PHOTOS API URL NOT DEFINED');
+  }
 
-	try {
-		const accessToken = obtainToken('newToken');
+  try {
+    const accessToken = obtainToken('newToken');
 
-		const photosSubmodule = new PhotosSubmodule({
-			baseUrl: photosUrl,
-			accessToken,
-		});
+    const photosSubmodule = new PhotosSubmodule({
+      baseUrl: photosUrl,
+      accessToken,
+    });
 
-		const { usage } = await photosSubmodule.getUsage();
+    const { usage } = await photosSubmodule.getUsage();
 
-		return usage;
-	} catch (error: any) {
-		Logger.warn('User is missing new access token, photos usage will not be counted');
-	}
+    return usage;
+  } catch (error: any) {
+    Logger.warn(
+      'User is missing new access token, photos usage will not be counted'
+    );
+  }
 
-	return 0;
+  return 0;
 }
 
-async function getDriveUsage(headers: HeadersInit): Promise<number> {
-	const response = await fetch(`${driveUrl}/api/usage`, {
-		headers,
-	});
-
-	if (!response.ok) {
-		Logger.error('Usage request failed');
-	}
+async function getDriveUsage(): Promise<number> {
+  const r = await httpClient.get(`${driveUrl}/api/usage`);
```

Comment:
This also exists in the SDK, maybe worth the refactor time

Get Drive Usage:
https://github.com/internxt/sdk/blob/c164a03d4778786046e4818be261222e7c438f21/src/drive/storage/index.ts#L246